packages/server/database/migrations/20210307124052-prepareBackfillTeams.ts

```diff
@@ -0,0 +1,33 @@
+export const up = async function (r) {
+  try {
+    await Promise.all([
+      r.db('actionProduction')
+        .table('Team')
+        .indexCreate('createdAt')
+        .run(),
+      r.db('actionProduction')
+        .table('Team')
+        .indexCreate('updatedAt')
```

Comment:
also need an indexWait call here otherwise the post migration could run before the index is ready (usually this isn't a problem, but we're making a back to back call on a pretty large table)